### DISCUSSION TITLE
Help to configure NginxProxyManager as Reverse proxy for SSL termination

### DISCUSSION THREAD

**Discussion by dcaccount** [NginxProxyManager/nginx-proxy-manager#3516] [Q&A]
Hello,
I need to configure NginxProxyManager as Reverse proxy for SSL termination. In particular NginxProxyManager has to act as reverse proxy for a Lighttpd web sever woking insecurely (no SSL) in the LAN.

It is not clear to me how to configure it. Lighttpd listens to port 80 but 80 is also mentioned in the docker-compose.yml file:

> ports:
      - '80:80'
      - '81:81'
      - '443:443'
      
and of course there cannot be two 80 ports listening at the same time on the host.

Can you please help me understand more and configure it as I would like?
Thanks,
Daniele

**Comment by bkeenke** [comment]
Let Lighttpd listen to 8080 if you can configure it, then in Nginx proxy manager you will add it to the “Proxy host”

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/128972800/ed642ab9-b1c3-43d3-b4a1-b52084ddb0a0)
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/128972800/e3f11162-e98a-432f-b51b-4c54a2480a0c)
